### PR TITLE
maplibre plugin interoperability

### PR DESCRIPTION
**Pull request by austyle**
New "news" blog post on AWS analysis and recommendations for developing plugin support and interoperability for both MapLibre and Mapbox.

### CODE REVIEW

**Comment by HarelM** [comment]
Nicely written, thanks!

**Comment by hy9be** [comment]
@HarelM @wipfli anything else we need to do before merging/publishing it?

**Comment by wipfli** [comment]
I fixed formatting issues with `npm run format`

**Review by wipfli — APPROVED** [review]
Thanks for the guest post, @austyle!

File: content/news/2023-01-03-aws-maplibre-plugins.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Developing Plugins for MapLibre Interoperability"
+date: "2023-01-04"
+categories: ["announcements"]
```
Comment (maxammann) [inline content/news/2023-01-03-aws-maplibre-plugins.md:4]: Change that to community-post

**Review by HarelM — APPROVED** [review]

File: content/news/2023-01-03-aws-maplibre-plugins.md (hunk 1 above)
Comment (wipfli) [inline content/news/2023-01-03-aws-maplibre-plugins.md:4]:
```suggestion
categories: ["community-post"]
```

**Pull request merged**